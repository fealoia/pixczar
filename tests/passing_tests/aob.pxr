Void main() {
  Int[] y= new Int[4];
  Int x = 4;
  y[x];
}

