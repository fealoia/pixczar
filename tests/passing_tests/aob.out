Throwing Runtime Error: Out of Bound Array Access
